README.md

```diff
@@ -15,20 +28,29 @@ The [Open Pediatric Cancer (OpenPedCan)](https://github.com/PediatricOpenTargets
     - [Small Variants](#small-variants)
     - [Copy Number Variants (CNVs)](#copy-number-variants-cnvs)
-- [RNA Sequencing](#rna-sequencing)
+- [RNA_Sequencing](#rna_sequencing)
   - [Data Processing](#data-processing-1)
   - [RNA Sequencing Data](#rna-sequencing-data)
     - [Fusions](#fusions)
     - [Gene Expression](#gene-expression)
     - [OpenPedCan Gene Expression Boxplot](#openpedcan-gene-expression-boxplot)
+- [Data_Availability](#data_availability)
+- [Citing the Molecular Targets Platform](#citing-the-molecular-targets-platform)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Datasets
 
-While adult pan-cancer repositories have existed and accelerated cancer research for a decade, pediatrics cancers have been excluded, despite having different genetic and molecular etiologies than adult cancers. Over the past few years larger pediatric consortia, both disease-specific and pan-cancer, have tried to address this disparity. The Molecular Targets Platform is harmonizing the data from across these different consortia in one unified location where it can be queried for associations between putative targets and pediatric cancers. As the project is ongoing, more data will continue to be added, but this current release includes 3 pediatric consortia datasets as well as GTEx data for comparisons to normal tissue expression:
+While adult pan-cancer repositories have existed and accelerated cancer research for a decade, pediatrics cancers have been excluded, despite having different genetic and molecular etiologies than adult cancers. 
+Over the past few years larger pediatric consortia, both disease-specific and pan-cancer, have tried to address this disparity. 
+OpenPedCan is harmonizing the data from across these different consortia and presenting summarized data in one unified location in the pediatric MTP where it can be queried for associations between putative targets and pediatric cancers. 
+As the project is ongoing, more data will continue to be added, but this current release includes four pediatric consortia datasets as well as GTEx data for comparisons to normal tissue expression:
+
+![Image of a table giving the count of biospecimens for each sequencing type (DNA or RNA) for each tumor stage (primary or relapse) for each Dataset](figures/biospecimen_dataset_counts.png)
 
-![Image of a table giving the count of biospecimens for each sequencing type (DNA or RNA) for each tumor stage (primary or relapse) for each Dataset](figures/biospecimen_counts.png)
+Primary tumors follow the standard definition of the original tumor in the body. 
+However, for analysis purposes, MTP combines progressive, progressive post-mortem, and relapse tumors into our relapse tumor stage.
+If needed, more detailed sample information is available in the histology data provided through the [OpenPedCan-analysis GitHub repository](https://github.com/PediatricOpenTargets/OpenPedCan-analysis#how-to-obtain-openpedcan-data).
```

Comment:
Added a relapse description as mentioned in https://github.com/PediatricOpenTargets/documentation/pull/8#pullrequestreview-1200299378. How does this sounds?

Reply:
perfect!